docs/cep/query-guide/functions/geospatial/crosses.md

```diff
@@ -24,25 +24,25 @@ Determines if the specified object or location crosses a geographic location spe
 ## Example 1
 
 ```sql
-CREATE STREAM StreamA (attribute_0 double, attribute_1 double, attribute_2 double, attribute_3 double);
-CREATE SINK STREAM OutputStream (attribute_0 double, attribute_1 double, attribute_2 double, attribute_3 double, prediction double, confidence double);
-
-@info(name = 'basicBayesianRegressionQuery')
-FROM StreamA#streamingml:bayesianRegression('model1', attribute_0, attribute_1, attribute_2, attribute_3)
-INSERT ALL EVENTS INTO OutputStream;
+SELECT geo:crosses('km-4354', -0.5, 0.5, '{"type":"Polygon","coordinates":[[[0, 0],[2, 0],[2, 1],[0, 1],[0, 0]]]}') AS crosses;
```

Comment:
@diana-macrometa - This is not a query technically but part of it. I would suggest to get rid of `SELECT` and `AS crosses`.
The below you could get rid of `This query uses`.

Reply:
My thought with this syntax is that we could give the user a line they could copy and paste into their own query.

Reply:
@diana-macrometa still you cannot do SELECT geo:crosses .. It has to be FROM InputStream#geo:crosses...

Reply:
Do I just need to change the SELECT to FROM?

Reply:
Yes SELECT to FROM InputStream#geo:crosses....